hcn/hcnsupport.go

```diff
@@ -18,6 +18,7 @@ type SupportedFeatures struct {
 	SetPolicy                bool        `json:"SetPolicy"`
 	VxlanPort                bool        `json:"VxlanPort"`
 	L4Proxy                  bool        `json:"L4Proxy"`
+	L4WfpProxy               bool        `json:"L4WfpProxy"`
```

Comment:
This could end up confusing to end users because they are very similar features (and both have WFP components). Can we just update the L4Proxy to be 13.2 and not have a second value?
Alternatively: can we rename the previous feature "L4ProxyOutbound"?
At the very least, let's add some comments to explain that the new thing handles inbound redirect and the previous dosent?

Reply:
Added a couple comments to distinguish these two